{"name": "Id курьера неверный", "status": "passed", "description": "Прием заказа с неверным ID курьера", "steps": [{"name": "принятие заказа ", "status": "passed", "start": 1726947210698, "stop": 1726947210853}, {"name": "Ожидаемый ответ 404, фактический 404", "status": "passed", "start": 1726947210853, "stop": 1726947210853}, {"name": "Ожидаемый json {'code': 404, 'message': 'Курьера с таким id не существует'}, фактический {'code': 404, 'message': 'Курьера с таким id не существует'}", "status": "passed", "start": 1726947210853, "stop": 1726947210853}], "start": 1726947210698, "stop": 1726947210854, "uuid": "4de72854-da1c-4c67-8654-195b9dca76d4", "historyId": "f42f0a6584bc37d48b3abe9aed8c4dd1", "testCaseId": "f42f0a6584bc37d48b3abe9aed8c4dd1", "fullName": "tests.test_order_accept.TestOrderAccept#test_accept_order_bad_id_courier", "labels": [{"name": "story", "value": "Прием заказа"}, {"name": "suite", "value": "Прием заказа"}, {"name": "subSuite", "value": "Негативные данные"}, {"name": "feature", "value": "Заказы"}, {"name": "parentSuite", "value": "tests"}, {"name": "host", "value": "KITCHEN"}, {"name": "thread", "value": "14296-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_order_accept"}]}